src/reporting/clients/IEC3ConfigurationsClient.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import { EC3Configuration, EC3ConfigurationCreate, EC3ConfigurationUpdate } from "../interfaces/EC3Configurations";
+import type { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IEC3ConfigurationsClient {
+  /**
+   * Gets all EC3 Configurations for an iModel. This method returns the full list of EC3 Configurations.
```

Comment:
EC3 configurations have nothing to do with iModels. They only refer to Reports. See https://developer.bentley.com/apis/carbon-calculation/operations/create-ec3-configuration/

Reply:
removed the ```for an iModel``` parts

Reply:
Is this comment resolved?

![image](https://user-images.githubusercontent.com/87982008/202456368-58df42dd-37b5-479e-ac4d-aefc9103bf6c.png)

Reply:
changed all iModel references to project references.